lib/file-helper.js

```diff
@@ -17,30 +21,25 @@ const cache = global.cache || {}
  * Directory listing for path
  *
  * @param {string} directoryPath
- * @returns {Promise<DirectoryListing>} Directory listing
+ * @param {string} [pattern] - Minimatch pattern
+ * @param {import('glob').IOptions} [options] - Glob options
+ * @returns {Promise<string[]>} File paths
```

Comment:
Thanks for making that update and going through all the changes for it  🙌🏻 🙌🏻 🙌🏻